src/gitmoji.js

```diff
@@ -88,7 +91,13 @@ class GitmojiCli {
     return this._fetchEmojis()
       .then((gitmojis) => prompts.gitmoji(gitmojis))
       .then((questions) => {
+        if (config.getSignedCommit() !== constants.NONE) {
+          questions = questions.filter(q => q.name !== 'signed')
+        }
         inquirer.prompt(questions).then((answers) => {
+          if (config.getSignedCommit() !== constants.NONE) {
```

Comment:
Same as the other block, you won't need it if you set the default value on the constructor 👍